guides/source/active_record_validations.md

```diff
@@ -1650,16 +1757,16 @@ irb> person.errors.empty?
 Displaying Validation Errors in Views
 -------------------------------------
 
-Once you've created a model and added validations, if that model is created via
-a web form, you probably want to display an error message when one of the
-validations fails.
+Once you've defined a model and added validations, you'll want to display an
+error message when a validation fails during the creation of that model via a
+web form.
 
-Because every application handles this kind of thing differently, Rails does
-not include any view helpers to help you generate these messages directly.
-However, due to the rich number of methods Rails gives you to interact with
-validations in general, you can build your own. In addition, when
-generating a scaffold, Rails will put some ERB into the `_form.html.erb` that
-it generates that displays the full list of errors on that model.
+Since every application handles this kind of thing differently, Rails does not
+include any view helpers to help you generate these messages directly. However,
+due to the rich number of methods Rails gives you to interact with validations
+in general, you can build your own. In addition, when generating a scaffold,
+Rails will put some generated ERB into the `_form.html.erb` that displays the
+full list of errors on that model.
```

Comment:
```suggestion
Since every application handles this kind of thing differently, Rails does not
include any view helpers to help you generate these messages directly. However,
due to the rich number of methods Rails gives you to interact with validations
in general, you can build your own. In addition, when generating a scaffold,
Rails will put some generated ERB into the `_form.html.erb` that displays the
full list of errors on that model.
```
```suggestion
Since every application handles displaying validation errors differently, Rails does not
include any view helpers to help you generate these messages. However, Rails gives you a rich number of methods to interact with validations in general that you can use to build your own. In addition, when generating a scaffold,
Rails will put some generated ERB into the `_form.html.erb` that displays the
full list of errors on that model.
```